Backend/app/rider-platform/rider-app/Main/src/Tools/Notifications.hs

```diff
@@ -464,12 +468,19 @@ notifyOnBookingCancelled booking cancellationSource bppDetails mbRide = do
             "Please book again to get another ride."
           ]
       SBCR.ByApplication ->
-        unwords
-          [ "Sorry your ride for",
-            showTimeIst (booking.startTime),
-            "was cancelled.",
-            "Please try to book again"
-          ]
+        case mbRide of
```

Comment:
Why if cancelled by Application, still we are saying ride is cancelled by driver?